implementation/contracts/system/FeeRebateToken.sol

```diff
@@ -3,6 +3,14 @@ pragma solidity ^0.5.10;
 import "openzeppelin-solidity/contracts/token/ERC721/ERC721Metadata.sol";
 import "./VendingMachineAuthority.sol";
 
+/// @title  Fee Rebate Token
+/// @notice The Fee Rebate Token (FRT) is a non fungible token (ERC721)
+///         the ID of which corresponds to a given deposit address. 
+///         If the corresponding deposit is still active, ownership of this token
+///         could result in reimbursement of the singer fee payed to open the deposit.
+/// @dev    This token is minted automatically when a TDT (TBTCDepositToken.sol)
+///         is exchanged for TBTC (TBTCToken.sol) via the 
+///         Vending Machine (VendingMachine.sol).
```

Comment:
Rather than referencing filenames here, I think you should just reference the contract names.

Reply:
maybe remove file names from everywhere?